docs/kb/semgrep-secrets/per-product-ignore-not-working.md

```diff
@@ -9,3 +9,5 @@ tags:
 The Semgrep AppSec Platform allows you to [define ignore patterns](https://semgrep.dev/docs/ignoring-files-folders-code#define-ignored-files-and-folders-in-semgrep-appsec-platform) for different Semgrep products for each project. Product-specific ignores for Semgrep Secrets require Semgrep version `1.71.0` or later in your CLI or CI environment.
 
 If you use an older version of Semgrep, the path ignores for Semgrep Secrets that are set in Semgrep AppSec Platform are not applied. Instead, the system applies the path ignores from SAST and SCA to your Secrets scan as well.
+
+Additionally, if you have [historical scanning](/semgrep-secrets/historical-scanning) enabled, the path ignores defined under Semgrep Secrets in the Semgrep AppSec Platform are not applied.
```

Comment:
Let's try to give some explanation about why, too. My understanding currently is that it's because we're scanning git blobs which aren't sensitive to path.